### ISSUE TITLE
Tab window scrolls to the top instead of the current tab

### ISSUE THREAD

**Issue by dabinat** [mozilla-mobile/firefox-ios#13810]
In the past two public releases I’ve experienced a problem where I tap the tab button to view all my tabs and the scroll bar is scrolled all the way to the top, instead of showing the tab that’s actually selected. I normally have quite a number of tabs open at once so it’s really slowed down my workflow by having to constantly scroll down to the currently selected tab every time I want to close it or switch to another tab.

Interestingly, if I scroll to a tab and open it, then open up the tabs window again immediately, the scroll bar is in the correct position. It’s only after the tab has been open for a certain amount of time that it ends up scrolled to the top, so it seems like Firefox is perhaps unloading that data in the background and when it recreates the window it doesn’t scroll it to the selected tab.

If it makes a difference, I mainly browse in private browsing mode. I am on iOS 16.4 and Firefox 111.2.

**Comment by dabinat** [comment]
Correction: I’m now not sure it’s related to the tab being open for a long time.

 I’m currently writing this in one tab; if I open the tabs window it’s scrolled to the top. I scroll down and pick the tab next to it to switch to. I then immediately hit the tabs button and it’s scrolled to the right place. I then go back to the GitHub tab, then hit the tabs button again and it’s scrolled to the top.

I am using an iPhone 8 so it could still be due to memory pressure.

**Comment by abodea** [comment]
Closing as duplicate of https://github.com/mozilla-mobile/firefox-ios/issues/13158.
